updatecli/values.yaml

```diff
@@ -11,3 +11,5 @@ jdks:
   - major: 11
   - major: 17
   - major: 21
+  - major: 25
+    releasetype: ea
```

Comment:
WDYT about tracking with updatecli on a second PR to make sure we do not miss anything with the first one?